### PR TITLE
Css beautifying

### PR DESCRIPTION
**Pull request by clfhhc**
added star review: https://trello.com/c/NXddvHHp/21-see-star-ratings

### CODE REVIEW

File: client/src/js/components/rates.jsx [hunk 1]
```diff
@@ -1,9 +1,85 @@
...
-  </div>
-);
+const starClassNames = {
+  containerClass: 'star-ratings room-rating',
+  fullStarClass: 'full-star room-rating',
+  pointFiveClass: 'point-five-star room-rating',
+  zeroStarClass: 'zero-star room-rating',
+  halfStarClass: 'half-star room-rating',
+  hiddenHalfStarClass: 'hidden-half-star room-rating',
+};
```
Comment (tonaramos) [inline client/src/js/components/rates.jsx]: What is the difference between point-five-class and half-star room-rating?

File: helpers/clientHelplers.jsx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+];
+
+export const getFullMonth = date => months[date.getMonth()];
+
+export const normalizeRating = (ratingRatio, fullRating, increment) => (
+  Math.ceil((ratingRatio * fullRating) / increment) * increment
+);
+
+/* eslint-disable react/no-array-index-key */
+export const makeStarElements = (
```
Comment (LoganAHart) [inline helpers/clientHelplers.jsx]: I like how you added svg to the project, look good!

File: client/src/css/header.css [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    flex-flow: column nowrap;
+    align-items: flex-start;
+    justify-content: center;
+    height: 146px;
+  }
+
+  .search-bar {
+    width: 100%;
+  }
+}
```
Comment (LoganAHart) [inline client/src/css/header.css]: Just a few newline's needed here and in one or two other files

File: client/src/css/reviews.css [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  text-align: left;
+}
+
+.half-star {
+  position: absolute;
+}
+
+.hidden-half-star {
+  position: absolute;
+}
```
Comment (LoganAHart) [inline client/src/css/reviews.css]: newline

File: client/src/css/rates.css [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+@media only screen and (max-width: 744px) {
+  .rates-list {
+    flex-direction: column;
+  }
+
+  .rates-column {
+    width: 100%;
+  }
+}
```
Comment (LoganAHart) [inline client/src/css/rates.css]: newline here as well

File: client/src/js/components/reviewListEntry.jsx [hunk 6]
```diff
@@ -1,9 +1,83 @@
...
+          ? truncateWords(this.props.review.text)
+          : null
+      ),
+    };
+    this.date = new Date(this.props.review.date);
+  }
+
+
+  render() {
+    console.log('something2');
```
Comment (tonaramos) [inline client/src/js/components/reviewListEntry.jsx]: left console.log here

File: client/src/js/components/reviewListEntry.jsx [hunk 7]
```diff
@@ -1,9 +1,83 @@
...
+                {this.state.truncatedText}
+                <span className="clickable read-more">Read more</span>
+              </span>
+            )
+            : (
+              <span>
+                {this.props.review.text}
+              </span>
+          )
+        }
```
Comment (tonaramos) [inline client/src/js/components/reviewListEntry.jsx]: this ternary matches the style guide, but just as a comment, to me example below makes more sense..
{ this.state.textTrundcated ?
    <>...</>
  ) : (
   <>...</>
  )
}

File: client/src/js/components/reviews.jsx [hunk 8]
```diff
@@ -20,17 +20,21 @@ const mapDispatchToProps = dispatch => ({
...
+    super(props);
+    this.timeout = null;
+  }
+
   componentDidMount() {
     this.props.selectARoom(this.props.roomId, this.props.numberReviewsPerPage);
   }
 
   render() {
+    console.log('something');
```
Comment (tonaramos) [inline client/src/js/components/reviews.jsx]: Left console.log here.

File: client/src/css/header.css [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+.search-bar.darker {
+  border-color: #484848;
+}
+
+.search-icon {
+  /* padding-top: 7px; */
```
Comment (rtsuzaki) [inline client/src/css/header.css]: remove

File: client/src/js/components/rates.jsx [hunk 10]
```diff
@@ -1,9 +1,85 @@
 import React from 'react';
+// import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { updateQuerySortBy } from '../actions/index';
+import { makeStarElements } from '../../../../helpers/clientHelplers';
```
Comment (rtsuzaki) [inline client/src/js/components/rates.jsx]: clientHelper is spelled wrong. please update file name

File: client/src/css/header.css (hunk 9 above)
Comment (clfhhc) [reply client/src/css/header.css]: ok, will incorporate in the next pr

File: client/src/css/header.css (hunk 3 above)
Comment (clfhhc) [reply client/src/css/header.css]: will incorporate in the next pr

File: client/src/js/components/rates.jsx (hunk 10 above)
Comment (clfhhc) [reply client/src/js/components/rates.jsx]: good catch!

File: helpers/clientHelplers.jsx (hunk 2 above)
Comment (clfhhc) [reply helpers/clientHelplers.jsx]: Thanks. Actually the svg is cloned from airbnb css files

**Pull request closed without merging**